src/drive/ducks/upload/index.js

```diff
@@ -76,11 +77,11 @@ const processNextFile = (
     // TODO: is the extractFileAttributes call really necessary?
     dispatch(fileUploadedCallback(extractFileAttributes(uploadedFile)))
   } catch (error) {
-    let status
-
-    if (error.status === 400) status = CONFLICT
-    else if (error.status === 413) status = QUOTA
-    else status = FAILED
+    const statusError = {
+      400: CONFLICT,
+      413: QUOTA
+    }
+    const status = statusError[error.status] || FAILED
```

Comment:
It's not blocking, but I'm not in favor of this change. It feels like we're going out of our way to not use `let`, making the mental gymnastic required to understand the code harder.

Reply:
I knew you'd say that, but I tried anyway. Here's my opinion:

In the original code, we had 6 instructions that can be the origin of 6 bugs.
In the refactored code, we only have 2 instructions: one object assignment (0 bug) and one value assignment with a well-known default value syntax. No `if`, no `else if` and the variable `status` is assigned only once.

That seems to me an easier code to maintain, to refactor, to understand and to debug.

Reply:
I could argue that there aren't 6 instructions in the original code, or that there are more than 2 in the new one, or explain the mental process when reading the old and new code… but I think we'll end up sitting here a long time, so let's not do that :)

I just wanted to point out that I don't think it's a good practice to refactor code that hasn't much to do with the current PR so that it suits our personal tastes. If we all start doing this, it's just going to be an endless edit war that won't benefit anyone ;)